test/functional/apps/discover/_shared_links.ts

```diff
@@ -80,7 +80,7 @@ export default function ({ getService, getPageObjects }: FtrProviderContext) {
             ":(from:'2015-09-19T06:31:44.000Z',to:'2015-09" +
             "-23T18:31:44.000Z'))&_a=(columns:!(),filters:!(),index:'logstash-" +
             "*',interval:auto,query:(language:kuery,query:'')" +
-            ",sort:!(!('@timestamp',desc)))";
+            ",rowsPerPage:100,sort:!(!('@timestamp',desc)))";
```

Comment:
it's recommendable not to add rowsPerPage to URL state if it's the default setting. given the default setting was changed, it would still stick to the old setting in this case